src/String/String.res

```diff
@@ -14,7 +14,10 @@ export concat = (str, append) => Js.String2.concat(str, append)
 %comment(
   "Returns `Some(value)` where `value` is the first character of the string, or `None` if the given string is empty."
 )
-export head = str => Js.String2.slice(str, ~from=0, ~to_=1)
+export head = str => {
+  let firstChar = Js.String2.slice(str, ~from=0, ~to_=1)
+  firstChar == "" ? None : Some(firstChar)
+}
```

Comment:
let's reuse the `get` function :) 

```rescript
export head = str => get(str, 0)
```

by the way, nice catch! 💪

Reply:
Great, I was wondering if I should use `get`